.ebextensions/eb-memory-monitor.config

```diff
@@ -1,24 +1,29 @@
 ###################################################################################################
 #### Memory monitoring for Elastic Beanstalk
-#### https://medium.com/tomincode/cloudwatch-memory-monitoring-for-elastic-beanstalk-1caa98d57d5c
+#### https://aws.amazon.com/premiumsupport/knowledge-center/elastic-beanstalk-memory-monitoring/
 ###################################################################################################
 
+packages:
+  yum:
+    perl-DateTime: []
+    perl-Sys-Syslog: []
+    perl-LWP-Protocol-https: []
+    perl-Switch: []
+    perl-URI: []
+    perl-Bundle-LWP: []
```

Comment:
From the docs:

Note: For Amazon Linux 2 based platforms, replace the perl-Bundle-LWP yum package in the preceding configuration file with perl-Digest-SHA.x86_64.

Should we add a TODO here tagged to a new issue so we remember